### ISSUE TITLE
Prints the error message: psutil._exceptions.TimeoutExpired: psutil.TimeoutExpired timeout after 3 seconds

### ISSUE THREAD

**Issue by yurivict** [nvbn/thefuck#741]
On FreeBSD I got this:
```
$ fuck
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/thefuck/output_readers/rerun.py", line 22, in _wait_output
    else settings.wait_command)
  File "/usr/local/lib/python3.6/site-packages/psutil/__init__.py", line 1264, in wait
    return self._proc.wait(timeout)
  File "/usr/local/lib/python3.6/site-packages/psutil/_psbsd.py", line 504, in wrapper
    return fun(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/psutil/_psbsd.py", line 760, in wait
    return _psposix.wait_pid(self.pid, timeout, self._name)
  File "/usr/local/lib/python3.6/site-packages/psutil/_psposix.py", line 104, in wait_pid
    delay = check_timeout(delay)
  File "/usr/local/lib/python3.6/site-packages/psutil/_psposix.py", line 66, in check_timeout
    raise TimeoutExpired(timeout, pid=pid, name=proc_name)
psutil._exceptions.TimeoutExpired: psutil.TimeoutExpired timeout after 3 seconds (pid=13634)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/thefuck", line 12, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/thefuck/entrypoints/main.py", line 25, in main
    fix_command(known_args)
  File "/usr/local/lib/python3.6/site-packages/thefuck/entrypoints/fix_command.py", line 36, in fix_command
    command = types.Command.from_raw_script(raw_command)
  File "/usr/local/lib/python3.6/site-packages/thefuck/types.py", line 82, in from_raw_script
    output = get_output(script, expanded)
  File "/usr/local/lib/python3.6/site-packages/thefuck/output_readers/__init__.py", line 18, in get_output
    return rerun.get_output(script, expanded)
  File "/usr/local/lib/python3.6/site-packages/thefuck/output_readers/rerun.py", line 47, in get_output
    if _wait_output(result, is_slow):
  File "/usr/local/lib/python3.6/site-packages/thefuck/output_readers/rerun.py", line 27, in _wait_output
    proc.kill()
  File "/usr/local/lib/python3.6/site-packages/psutil/__init__.py", line 286, in wrapper
    raise NoSuchProcess(self.pid, self._name)
psutil._exceptions.NoSuchProcess: psutil.NoSuchProcess process no longer exists (pid=13634)
```

**Comment by scorphus** [comment]
Can you please give us more information to reproduce this:

1. What was the command you were trying to correct? 
2. What is `thefuck --version` output?

Thanks!

**Comment by yurivict** [comment]
```
$ thefuck --version
The Fuck 3.25 using Python 3.6.2
```

I don't remember the command now. I will see if I can reproduce it.

**Comment by scorphus** [comment]
Fixed in current release, [3.29](https://pypi.org/project/thefuck/3.29/). Kindly update to it.
